### PR TITLE
update olm metadata for MAISTRA-1263

### PR DESCRIPTION
**Pull request by rcernich**
Signed-off-by: rcernich <rcernich@redhat.com>

### CODE REVIEW

**Review by dgn — APPROVED** [review]

File: manifests/1.1.0/maistraoperator.v1.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -352,7 +354,7 @@ spec:
                 - name: OPERATOR_NAME
                   value: istio-operator
                 - name: ISTIO_CNI_IMAGE_V1_0
-                  value: maistra/istio-cni-ubi8:1.0.0
```
Comment (gbaufake) [inline manifests/1.1.0/maistraoperator.v1.1.0.clusterserviceversion.yaml:355]: Should it be 1.08 or 1.0.9?

File: manifests/1.1.0/maistraoperator.v1.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -107,6 +108,7 @@ spec:
     url: https://github.com/Maistra/istio-operator
   - name: Istio
     url: https://istio.io/
+  replaces: maistraoperator.v1.0.8
```
Comment (gbaufake) [inline manifests/1.1.0/maistraoperator.v1.1.0.clusterserviceversion.yaml:111]: Should it be 1.09?

Comment (dgn) [reply manifests/1.1.0/maistraoperator.v1.1.0.clusterserviceversion.yaml:111]: Good thing you ask; I wondered the same at first but I forgot to comment.

We didn't release 1.0.9 in community, and this is only for community. See here: https://hub.docker.com/r/maistra/proxyv2-ubi8/tags

File: manifests/1.1.0/maistraoperator.v1.1.0.clusterserviceversion.yaml (hunk 1 above)
Comment (dgn) [reply manifests/1.1.0/maistraoperator.v1.1.0.clusterserviceversion.yaml:355]: See my comment below; there's no 1.0.9 in community

**Pull request merged**